admin_manual/configuration_server/config_sample_php_parameters.rst

```diff
@@ -2,38 +2,215 @@
+called ``$CONFIG``. This array consists of various Nextcloud specific "key-value" pairs (in some cases 
+arrays themselves). Each pair has the form ``key => value`` and is comma-separated.
+
+Types of Values
+^^^^^^^^^^^^^^^
+
+Strings: 
+
+* ``"thisIsAnImportantValue"``
+* Note: These must be either single or double quoted - i.e. ``"string"`` or ``'string'``.
+* Note: IP addresses are considered strings.
+* Examples:
+   - ``'logo_url' => 'https://example.org',``
+   - ``'versions_retention_obligation' => 'auto, D',``
+   - ``'logtimezone' => 'Europe/Berlin',``
+
+Boolean: 
+
+* ``true`` or ``false``
+* Note: These should **not** be surrounded by quote marks within the configuration file itself.
+* Examples:
+   - ``'session_keepalive' => true,``
+   - ``'hide_login_form' => false,``
+
+Numerical:
+
+* ``12``
+* This includes both integers and floating point numbers.
+* Note: These should **not** be surrounded by quote marks within the configuration file itself.
+* Examples:
+   - ``'loglevel' => 2,``
+   - ``'session_lifetime' => 60 * 60 * 24,``
+
+Arrays of any of the above types:
+
+* ``[ 'value1', 'value2' ]``
+* All value types (including other arrays) can be included in arrays.
+* Note: Only some parameters support array style values.
+* Examples:
+   - ``'connectivity_check_domains' => [ 'www.nextcloud.com', 'www.eff.org', ],``
+   - ``'enabledPreviewProviders' => [ 'OC\Preview\BMP', 'OC\Preview\GIF', 'OC\Preview\JPEG', ],``
+
+.. tip:: Nextcloud attempts to remedy some value type/formatting mistakes, but this is not foolproof. 
+   Use the correct formatting (for the type of value in question) to avoid unexpected results arising 
+   from values being cast in unexpected ways.
+
+Modifying
+---------
+
+Parameters may be modified in a standard text editor (i.e. via the command-line or externally 
+then re-uploaded). They may also, in most cases, be modified using the commands in
+the ``occ config:system:*`` namespace.
+
+.. tip:: Incorrectly formatted ``key => value`` entries (or incorrectly specified values) may
+   not generate immediate errors or problems (such as parsing / syntax errors), but may still 
+   lead to unexpected and undesirable results. Review your fully parsed (by PHP) configuration
+   by using the command ``occ config:list system`` and/or ``occ config:list system --private``
+   to identify anything unexpected.
+
+Defaults
+--------
+
+Nextcloud creates a base ``config/config.php`` file at installation time containing the most 
+essential parameters for operations. These values are a mixture of auto-generated and drawn from
+information provided by the administrator at installation time.
+
+The file ``config/config.sample.php`` lists all the parameters within Nextcloud that can be 
+specified in ``config/`` files, along with example and default values for each. The content of 
+that sample configuration file is included here for ease of reference and alongside additional 
+context.
```

Comment:
Since it is some sections further, maybe add a link?
```suggestion
that sample configuration file is listed in the `sections below <#user-experience>`_ for ease of reference and
alongside additional context.
```

Reply:
Good call. Done. Added a stable id to use with a a `:ref:` to the config.php.sample section.